### PR TITLE
doc: remove spurious console code block from arduino_zero doc.

### PR DESCRIPTION
**Pull request by henrikbrixandersen**
Signed-off-by: Henrik Brix Andersen <henrik@brixandersen.dk>

### CODE REVIEW

File: boards/arm/arduino_zero/doc/arduino_zero.rst [hunk 1]
```diff
@@ -120,8 +120,6 @@ Flashing
 
 #. To flash an image:
 
-   .. code-block:: console
-
 	.. zephyr-app-commands::
```
Comment (dbkinder) [inline boards/arm/arduino_zero/doc/arduino_zero.rst]: You should also outdent this ``.. zephyr-app-commands::`` directive too (align with the T in "To flash")
(the same for the directive on line 97 above)

**Review by dbkinder — CHANGES_REQUESTED** [review]
needs a body in the commit message.  Also, while you're in there can you fix the indent of the two zephyr-app-commands directive.

**Comment by henrikbrixandersen** [comment]
@dbkinder Thanks. I have updated the commit to fix the indentation of the two zephyr-app-commands directives and added a commit message body. Please review.

**Review by nashif — APPROVED** [review]

**Review by dbkinder — APPROVED** [review]
Thanks

**Pull request merged**